odo/backends/tests/test_url.py

```diff
@@ -34,6 +34,19 @@ def test_url_resource():
     assert isinstance(csv, URL(CSV))
 
 
+def test_small_chunk_size():
+    url_csv = resource(iris_url)
+    normal = convert(Temp(CSV), url_csv)
+    url_csv.chunk_size = 1
```

Comment:
can we pass this as a kwarg to `resource` or is it expected to mutate the object after creation?